kotlin-extensions/src/main/java/com/amplifyframework/ktx/DataStoreCategoryExt.kt

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@file:Suppress("unused")
+
+package com.amplifyframework.ktx // TODO: or .datastore?
```

Comment:
Yeah I don't love `ktx` for some reason.

You could just use the original package name since these are extensions to those models or perhaps `coroutines`. 

All just bike shedding though.

Reply:
^ As I've thought about this more, I don't think it makes sense to make a separate library for this work. I can add these extensions directly to our `core` Java library. Our existing APIs will continue to work, but people who are using Kotlin will automatically get the goodness.

Reply:
The only downside to that is the additional dependency/dependencies introduced (I'm guessing that's `coroutines` and `kotlin-stdlib`?).

Reply:
^ I keep going back and forth on this. The current version of the PR keeps the extensions as a separate module, mostly because of the new dependencies. The Android platform also tends to segregate out its Kotlin extension libraries. So, that's probably a good way to go for cultural consistency.

Reply:
I ended up changing this to `.kotlin`.